README.md

```diff
@@ -22,7 +22,9 @@ The scripts and documentation in this project are released under the [MIT Licens
 
 ## Development
 
-Once the dev container is started do the following:
+Use [Visual Studio Code](https://code.visualstudio.com/) with the [Remote Development extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack).
```

Comment:
FYI the extension will be mentioned automatically by VS Code. So technically, there is no requirement to preinstall it